### ISSUE TITLE
Support setting an embargo

### ISSUE THREAD

**Issue by LynnMcRae** [sul-dlss/argo#1051]
We currently have an "Update Embargo" blue button to change existing embargo settings. We lack the ability to add an embargo in the first place.  Suggest blue button could change to "Manage Embargo" and updated to support creating or changing embargo.

blocked by sul-dlss/dor-services#481

Requires some new UI, since setting of an embargo requires the user specify both a date and new rights to change to after embargo period passes

**Comment by andrewjbtw** [comment]
Closing in favor of https://github.com/sul-dlss/argo/issues/2009
